R/allocation_data_cleaning.R

```diff
@@ -0,0 +1,48 @@
+# Author: Paigan Aspinall
+# Date: 20SEP2023
+# R version: 4.2.2
+
+library(dplyr) #1.1.0
```

Comment:
Don't think you need this line, I think these are now all in the package imports so may not be required in each file.